src/Selection.js

```diff
@@ -317,6 +317,9 @@ class Selection {
   }
 
   _handleMoveEvent(e) {
```

Comment:
IE was throwing errors on this `let { x, y } = this._initialEventData` if `this._initialEventData` is null